index.js

```diff
@@ -5,4 +5,199 @@ function showHide() {
 const menuBtn = document.querySelector('#menuBtn');
 menuBtn.addEventListener('click', showHide);
 const xBtn = document.querySelector('#xBtn');
-xBtn.addEventListener('click', showHide);
\ No newline at end of file
+xBtn.addEventListener('click', showHide);
+
+//  popUp
+
+const projects = [
+  {
+    name: 'Keeping track of hundreds  of components website',
+    img: './Assets/ex.png',
```

Comment:
Good job so far 💯 in the other examples I had to adjust the link to see the design correctly, the reason being that your link here is broken ❌ 😞  take a look 👇🏽 

![Screenshot 2023-04-12 at 22 23 26](https://user-images.githubusercontent.com/45739963/231631100-8082d279-e349-4061-ae66-92f7104d7188.png)

Reply:
but it is working in my local envirement, I could not understand the reasen !

Reply:
Hi @OTMANE-ECHCHAFYKY , Check the img naming in your array